src/screens/Learning.js

```diff
@@ -133,7 +143,11 @@ export default ({
           </View>
           <View style={styles.heading}>
             <SectionHeading text="Category: " />
-            <Text>{currentCategoryName}</Text>
+            <Text>{`${
+              learntPhrasesCat
```

Comment:
as discussed we dont need 'learntPhrasesCat' in redux state try to set current category when clicking on the main screen. Set it to id ###learnt-phrases###
Then pass the existing prop current cat id from state here and do the condition logic based on that.